### ISSUE TITLE
[MAS2.1.1] Unable to scroll 'Inspector' and 'Log' sections using keyboard.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#742]
**Actual Result:**
User is not able to scroll Inspector and Log sections using keyboard and hence not able to see the complete content present in these sections.

**Expected Result:**
User should be able to scroll Inspector and Log sections using keyboard and hence should be able to see the complete content present in these sections.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Click on open bot and choose the .bot file that is available in the bot folder.
3. Verify that live chat tab is open.
4. Notice Inspector and Log sections present in right side.
5. Click on any chat text using mouse and notice that Information related to this chat text line is appearing in the Inspector section.
6. Navigate to 'Inspector' and 'Log' sections one by one and try to scroll content present in these sections using keyboard.
 
**User Impact:**
Keyboard dependent users won't be able to use the controls which are not accessible using keyboard in any way.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link:**
[MAS 2.1.1 – Keyboard (36)
](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

**Attachment:**
[MAS2.1.1_BingFrameworkEmulator_InspectorLog_Sections_Scroll.zip](https://github.com/Microsoft/BotFramework-Emulator/files/2266509/MAS2.1.1_BingFrameworkEmulator_InspectorLog_Sections_Scroll.zip)
